docker/web/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+MAINTAINER Pedro Marques <roque@juniper.net>
+RUN apt-get install -y software-properties-common
+RUN apt-add-repository -y ppa:anantha-l/opencontrail-2.20
+RUN apt-get update
+RUN apt-get -y install gdebi wget
+
+RUN wget -qO /root/nodejs_0.8.15-1contrail1_amd64.deb https://launchpad.net/~opencontrail/+archive/ubuntu/ppa/+files/nodejs_0.8.15-1contrail1_amd64.deb
```

Comment:
apt-get gets a different version with which our web code did not work at
the time. A related bug is
https://bugs.launchpad.net/opencontrail/+bug/1395127


On Wed, Oct 28, 2015 at 4:39 AM, Pedro Marques <notifications@github.com>
wrote:

> In docker/web/Dockerfile
> <https://github.com/pedro-r-marques/examples/pull/1#discussion_r43200708>:
>
> > @@ -0,0 +1,29 @@
> > +FROM ubuntu:14.04
> > +MAINTAINER Pedro Marques <roque@juniper.net>
> > +RUN apt-get install -y software-properties-common
> > +RUN apt-add-repository -y ppa:anantha-l/opencontrail-2.20
> > +RUN apt-get update
> > +RUN apt-get -y install gdebi wget
> > +
> > +RUN wget -qO /root/nodejs_0.8.15-1contrail1_amd64.deb https://launchpad.net/~opencontrail/+archive/ubuntu/ppa/+files/nodejs_0.8.15-1contrail1_amd64.deb
>
> Why can't you do apt-get download ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pedro-r-marques/examples/pull/1/files#r43200708>.
>